jwst/combine_1d/combine_1d_step.py

```diff
@@ -17,17 +18,17 @@ class Combine1dStep(Step):
     spec = """
     # integration_time or exposure_time.
     exptime_key = string(default="exposure_time")
-    # Interpolation between pixels.
-    interpolation = string(default="nearest")
+    # This is currently not used.
+    interpolation = string(default="")
```

Comment:
Would it make sense to set the default for `interpolation` to `None`?  If so, then it should be removed from the `combine_1d.cfg` file too.